### PR TITLE
post: ng 7.0

### PR DESCRIPTION
**Pull request by cexbrayat**
Small release, so short blog post.

Todo:
- [x] change date
- [x] link to changelog
- [x] check if https://github.com/angular/angular/pull/25839 landed
- [ ] add a link to CLI post

### CODE REVIEW

File: _posts/2018-10-19-what-is-new-angular-7.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+It will not affect you,
+but should fix a bunch of issues when multiple navigations were triggered at the same time
+and will allow new features more easily in the future.
+
+## Deprecations
+
+As it's usually the case with major releases, a few things has been deprecated.
+If you are using `<ngForm>` to declare a form in your template
+(you don't have to, as `form` also activates the NgForm directive),
+this selector is now deprecated and should be by replaced `<ng-form>`.
```
Comment (clacote) [inline _posts/2018-10-19-what-is-new-angular-7.md]: by replaced -> replaced by

File: _posts/2018-10-18-what-is-new-angular-7.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  </a>
+</p>
+
+Not a lot of new features in this release:
+the Angular team is mainly focused on the Ivy project,
+rewriting the Angular compiler and runtime code to make it smaller, better, faster.
+But Ivy is not ready for prime time yet.
+
+So don't expect a lot of shiny things in Angular 7.0:
+there was not enough material to make a video as we did for [Angular 5](/2017/11/02/what-is-new-in-angular-5) or [Angular 6](/2018/05/04/what-is-new-in-angular-5).
```
Comment (clacote) [inline _posts/2018-10-18-what-is-new-angular-7.md]: Wrong url for Angular 6 -> /2018/05/04/what-is-new-angular-6/

File: _posts/2018-10-18-what-is-new-angular-7.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+The rewrite is making progress, but Ivy is still not usable in this release.
+If you refer to the [official feature tracking](https://github.com/angular/angular/blob/master/packages/core/src/render3/STATUS.md),
+a good chunk of the work is done.
+But in reality, there is still a long way to go before we can use it.
+
+Ivy has several pieces:
+
+- `ngtsc`: the compiler that compiles your Angular application and generates JavaScript from your HTML templates. This piece of code has made good progress but still misses a few features.
+- `ngcc`: a tool that explores all the dependencies you have, to convert existing code into "Ivy compatible" code. This is still very early stage, and barely usable at the moment if you don't know how to workaround a few issues
```
Comment (clacote) [inline _posts/2018-10-18-what-is-new-angular-7.md]: "This is still very early stage" -> This is still **at** very early stage, no?

File: _posts/2018-10-18-what-is-new-angular-7.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+But you can check it out yourself, as the CLI added an `--experimental-ivy` flag
+to generate an application with the configuration needed to try it.
+
+    ng new ivy-test --experimental-ivy
+    cd ivy-test
+    $(npm bin)/ngcc
+    ng serve --aot
+
+Note that the change detection is not working when I'm writing these lines,
```
Comment (clacote) [inline _posts/2018-10-18-what-is-new-angular-7.md]: when I'm writing -> as I'm writing, no?

File: _posts/2018-10-18-what-is-new-angular-7.md (hunk 3 above)
Comment (cexbrayat) [reply _posts/2018-10-18-what-is-new-angular-7.md]: I think it is correct too...

File: _posts/2018-10-18-what-is-new-angular-7.md (hunk 2 above)
Comment (cexbrayat) [reply _posts/2018-10-18-what-is-new-angular-7.md]: Bien vu!

**Pull request merged**